Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:buster-slim
+RUN apt-get update -y && apt-get install ca-certificates -y
+RUN update-ca-certificates
+ADD https://github.com/gohugoio/hugo/releases/download/v0.69.0/hugo_extended_0.69.0_Linux-64bit.deb /tmp
+RUN dpkg -i /tmp/hugo_extended_0.69.0_Linux-64bit.deb
+ENTRYPOINT hugo
```

Comment:
Jeszcze gita trzeba zainstalować:

```suggestion
RUN apt install -y git
ENTRYPOINT hugo
```

Reply:
Wydaje mi się, że nie :) debian:buster-slim musi mieć gita. Autor tego nic nie wspomina.

Reply:
Chyba jednak trzeba. Korzystałem z tego repo jako akcji, a tam taki sam Dockerfile jak tutaj:
https://github.com/lowply/build-hugo/blob/master/Dockerfile
Przy wykonaniu brakowało gita, ale to dlatego, że szablon LoveIt go używa do sprawdzania w jakiej rewizji były ostatnie zmiany i wyświetla tę informację na stronie.
![Zrzut ekranu 2020-04-22 o 09 55 19](https://user-images.githubusercontent.com/4595135/79955695-719c4080-847f-11ea-94e2-8fe921660ddb.png)

Reply:
Info o błędzie: https://github.com/coders-school/site/runs/597813668?check_suite_focus=true